lib/channel_model.js

```diff
@@ -74,10 +98,16 @@ C.open = function() {
 
 C.close = function() {
   var self = this;
-  return Promise.fromCallback(function(cb) {
+
+  return Bluebird.fromCallback(function(cb) {
```

Comment:
Am I misunderstanding something, or promisify could be used here instead of Bluebird?

Reply:
After dropping old Node version — it could

Reply:
Only Node 10+ is supported in master, so should be fine to use it now.

Reply:
Replaced with util.promisify

Reply:
btw .travis.yml in main still builds library for old Node.js versions

Reply:
Yes, I think it was just missed in https://github.com/squaremo/amqp.node/commit/267dd4e17c93efc755e40a4645510fdf70b9ebb8. Probably Travis needs to be adjusted to only test for Node 10+ as well.